lib/grids_page.dart

```diff
@@ -82,8 +87,10 @@ class SelectGrid extends StatelessWidget {
       cursor: SystemMouseCursors.click,
       child: GestureDetector(
         onTap: () {
-          currentGrid.value = grid;
-          Navigator.pop(context);
+          if (grid.length > boardSize) {
```

Comment:
Can you add comment why we need this condition?

Reply:
Sorry, we don't need it, I thought to make changing of avaliable in-a-row depending of the board size, but made it easier way

Reply:
Deleted.